scripts/import/import_EPMC.pl

```diff
@@ -137,9 +137,10 @@ sub import_citations{
     my $data           = shift;
     my $type           = shift;
 
-    ## store variants not from this species
     open my $error_log, ">>$species\_$type\_notfound_" . log_time() . "\.log"|| die "Failed to open log file: $!\n";
-
+    ## store variants not from this species
+    open my $not_found, ">>rsID\_$type\_notfound\.csv"|| die "Failed to open file: $!\n";
```

Comment:
Without the species in the file name, the variants not found in one species will overwrite the variants not found in another species if the jobs are run in the same directory. Running 2 species at once becomes complex.
 
Without the timestamp, there is also a risk of over-writing.